pages/jobs/[slug].tsx

```diff
@@ -30,6 +31,7 @@ const JobPosting: NextPage<JobPageProps> = ({ preview, job, content }) => {
 
     return (
         <Layout theme="LIGHT" preview={preview} jobsAvailable={8}>
+            <SchemaGenerator job={job} />
```

Comment:
This might need some refactoring, currently it looks like this is being rendered inside body tags. there can only be one head element which already looks to be included in `MyApp`. head elements should also be outside the body tag.

Reply:
Ignore me, `import Head from 'next/head';` handles this.